docs/source/en/quantization/torchao.md

```diff
@@ -45,7 +45,9 @@ pipe = FluxPipeline.from_pretrained(
 pipe.to("cuda")
 
 prompt = "A cat holding a sign that says hello world"
-image = pipe(prompt, num_inference_steps=28, guidance_scale=0.0).images[0]
+image = pipe(
+    prompt, num_inference_steps=50, guidance_scale=4.5, max_sequence_length=512
```

Comment:
Dev is a guidance-distilled model and as such we cannot have `guidance_scale=0.0`.